src/pages/workspace/WorkspaceMembersPage.js

```diff
@@ -299,6 +328,18 @@ class WorkspaceMembersPage extends React.Component {
             });
         });
         data = _.sortBy(data, value => value.displayName.toLowerCase());
+        const searchValue = this.state.searchValue.trim().toLowerCase();
+        data = _.filter(data, (member) => {
+            if (this.isKeywordMatch(member.displayName, searchValue)
+                || this.isKeywordMatch(member.login, searchValue)
+                || this.isKeywordMatch(member.phoneNumber, searchValue)
+                || this.isKeywordMatch(member.firstName, searchValue)
+                || this.isKeywordMatch(member.lastName, searchValue)) {
+                return true;
```

Comment:
This conditional can get inline instead of explicitly returning true / false.

Reply:
```suggestion
                data = _.filter(data, member => this.isKeywordMatch(member.displayName, searchValue) || this.isKeywordMatch(member.login, searchValue) || this.isKeywordMatch(member.phoneNumber, searchValue) || this.isKeywordMatch(member.firstName, searchValue) || this.isKeywordMatch(member.lastName, searchValue));
```

@marcaaron i have set condition inline but lint giving warning `This line has a length of 307. Maximum allowed is 190`. if its ok then i can commit this change?

Reply:
yes @0xmiroslav i try to put them multiple lines but lint don't allow it and error is `error  Trailing spaces not allowed `

Reply:
@marcaaron if its not necessary can we skip this optimization?

Reply:
I think we can break it out into multiple lines. If you have trailing spaces that sounds like a different issue.